sql/core/src/test/resources/sql-functions/sql-expression-schema.md

```diff
@@ -0,0 +1,341 @@
+<!-- Automatically generated byExpressionsSchemaSuite -->
```

Comment:
Why is this file md specifically?

Reply:
See https://github.com/apache/spark/pull/28194#issuecomment-613854769

I guess it's easier for humans the read the table.

Reply:
Okay, but seems it's more difficult to track the diff. See https://amplab.cs.berkeley.edu/jenkins/job/spark-branch-3.0-test-sbt-hadoop-2.7-hive-2.3/431/testReport/junit/org.apache.spark.sql/ExpressionsSchemaSuite/Check_schemas_for_expression_examples/

![Screen Shot 2020-05-01 at 2 06 10 PM](https://user-images.githubusercontent.com/6477701/80783491-f5a7a400-8bb4-11ea-9e91-880719bd472c.png)
![Screen Shot 2020-05-01 at 2 06 04 PM](https://user-images.githubusercontent.com/6477701/80783494-f7716780-8bb4-11ea-9425-dfc348644649.png)
![Screen Shot 2020-05-01 at 2 05 58 PM](https://user-images.githubusercontent.com/6477701/80783495-f8a29480-8bb4-11ea-8b7a-eb16cbcc9a8e.png)
![Screen Shot 2020-05-01 at 2 05 48 PM](https://user-images.githubusercontent.com/6477701/80783498-f9d3c180-8bb4-11ea-82b4-6f865294a21f.png)
![Screen Shot 2020-05-01 at 2 05 42 PM](https://user-images.githubusercontent.com/6477701/80783501-fa6c5800-8bb4-11ea-9ce6-f562aebf6b6c.png)
![Screen Shot 2020-05-01 at 2 05 35 PM](https://user-images.githubusercontent.com/6477701/80783504-fb9d8500-8bb4-11ea-9a9a-f0e5eadcb596.png)

Reply:
Hopefully we can improve the diff here ...

Reply:
This should be row-by-row diff, @beliefer can you help to fix it?

Reply:
@cloud-fan @HyukjinKwon 
This checked exception is thrown when the number of expected SQL and the number of actual SQL are not equal.
Could I not output all the SQL to the checked exception ?